<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExplorerFrame</class>
 <widget class="QFrame" name="FileExplorerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="3" column="0">
    <widget class="QPushButton" name="pbRemove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="bcPath" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Prova</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Due</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbAdd">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="MDBreadCrumb" name="breadCrumb">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDBreadCrumb</class>
   <extends>QListWidget</extends>
   <header>gui/custom-widgets/mdbreadcrumb.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
